site/learn/expert-advice/identity-basics/registration-best-practices.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: advice
+title: Best practices for registration forms
+description: Users have to register to gain application access. How can you make this as painless as possible?
```

Comment:
"Users have to fill out registration forms in order to gain application access"... or some way to work in 'registration forms'